components/dashboard/src/data/usage/usage-query.ts

```diff
@@ -15,8 +15,8 @@ export function useListUsage(request: ListUsageRequest) {
             return getGitpodService().server.listUsage(request);
         },
         {
-            cacheTime: 1000 * 60 * 10, // 10 minutes
-            staleTime: 1000 * 60 * 10, // 10 minutes
+            cacheTime: 1000 * 60 * 1, // 1 minutes
```

Comment:
The reason data is not updated every 15 minutes is because we have dashboard caches, not sure if we do want to remove caches, 1m should be fine for users